lv_conf_template.h

```diff
@@ -439,6 +439,8 @@
 
 #define LV_USE_CHECKBOX   1
 
+#define LV_USE_COLORWHEEL 1
```

Comment:
The color wheel was removed.

Reply:
Sure, removing other mentions of COLORWHEEL